GTG/gtk/browser/main_window.py

```diff
@@ -1295,7 +1295,8 @@ def _reapply_filter(self, current_pane):
         # whether to refilter or not
         if sorted(filters) != sorted(vtree.list_applied_filters()):
             vtree.reset_filters(refresh=False)
-        # browsing and applying filters
+        # Browsing and applying filters. For performance optimization, only
+        # allowing liblarch to trigger a refresh on last item.
```

Comment:
Doesn't really say why you couldn't just do `refilter`. Maybe append "This way, liblarch can prevent refreshing in certain situations rather than refilter all the time."

Reply:
Ok I'll edit that. But truth is, I can't call on `ViewTree.refilter()` because there is no such method. `ViewTree` doesn't proxy that method to `FilteredTree` and I don't know why at all. This is what I meant in the second part of that [comment](https://github.com/getting-things-gnome/gtg/pull/530#discussion_r554457225).